acc.php

```diff
@@ -2035,6 +2035,43 @@
 		}
 	});
 }
```

Comment:
Minor code style: elseif should be on the same line as the closing brace of the previous block